stream-chat-android-client/src/main/java/io/getstream/chat/android/client/channel/ChannelClientExtensions.kt

```diff
@@ -1,4 +1,4 @@
-package io.getstream.chat.android.client.controllers
+package io.getstream.chat.android.client.channel
```

Comment:
This is a breaking change for clients. It's very new and fancy API though, so it might be okay, if we document it in the changelog.